stories/Progress/Progress.stories.js

```diff
@@ -115,8 +115,7 @@ const EsempiInterattiviComponent = () => {
                   <div className="progress-bar-label">
                         <span className="sr-only">Progresso </span>{value}%
                   </div>
-
-                  <div className="progress progress-color" style={{marginBottom: "2em", height:`${height}px`}}>
+                  <div className="progress progress-color">
                         <div className={`progress-bar bg-${color}`} role="progressbar" style={{width: `${value}%`}} aria-valuenow={`${value}`} aria-valuemin="0" aria-valuemax="100">{label}</div>
```

Comment:
Yup! looks good. As far as Label or Etichette is considered in the Interactive component, it is not displayed due to the height of progress bar. You may merge it and I will fix it and add other progress indicators.